src/config.yml

```diff
@@ -66,7 +66,7 @@ allow:
 #
 # * The second section is a letter which indicates what to do when a player executes that command.
 #   Valid actions:
-#   b - Block the command
+#   b - Block the command for everyone 
```

Comment:
No. _b_ does not block the command for everyone. It's an action, not a player-selector. _n_ is the player selector you mean.